src/agent/state.js

```diff
@@ -290,7 +296,16 @@ StateResolver.prototype.isPathInNodeModulesDirectory_ = function(path) {
 StateResolver.prototype.resolveFrame_ = function(frame, underFrameCap) {
   var args = [];
   var locals = [];
-  if (!underFrameCap) {
+  if (!this.config_.allowExpressions) {
+    args.push({
+      name: 'arguments_not_available',
+      varTableIndex: EXPRESSIONS_DISALLOWED_INDEX
+    });
+    locals.push({
+      name: 'locals_not_available',
```

Comment:
I remember talking through it when we talked about the behavior of the new option being added but I don't recall the reason. We already avoid interceptors/getters on objects by default so object properties should be ok if locals are in general?